packages/ui/src/molecules/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -60,6 +60,39 @@ describe('Breadcrumb', () => {
     }
   })
 
+  it('renders an <ol> with valid HTML as children (only <li>s)', () => {
+    const { getByRole } = render(
+      <Breadcrumb divider="/">
+        <a href="/office">Office</a>
+        <a href="/office/chairs">Chairs</a>
+      </Breadcrumb>
+    )
+
+    const listItems = getByRole('list').children
+
+    for (const item of Array.from(listItems)) {
+      expect(item.nodeName).toEqual('LI')
+    }
+    expect(listItems).toHaveLength(2)
```

Comment:
`GitHub Actions / Lint` is complaining: 

> Expected blank line before this statement

```suggestion

    expect(listItems).toHaveLength(2)
```

Reply:
Ops, I missed that! Fixed that in this other PR https://github.com/vtex/faststore/pull/1159, please take a look!